packages/styleUtils/modifiers/stories/Box.stories.tsx

```diff
@@ -57,88 +48,91 @@ export default {
   }
 } as Meta;
 
-const Template: Story<BoxProps> = args => (
-  <Box {...args}>
-    <div
-      style={{
-        border: `2px solid ${themeBrandPrimary}`,
-        height: "300px",
-        textAlign: "center"
-      }}
-    >
-      Box
-    </div>
-  </Box>
-);
+const setHeight = css`
+  > * {
+    height: 200px;
+  }
+`;
+
+const Template: Story<BoxProps> = args => {
+  const addBorder = css`border: 2px solid ${themeBrandPrimary}`
```

Comment:
🥜  Might be best to move this border outside of the template. Or you can pass in the emotion css inline` <div className={css``border: 2px solid ${themeBrandPrimary}``}>`